UefiPayloadPkg/UniversalPayloadBuild.py

```diff
@@ -256,7 +256,7 @@ def BuildUniversalPayload(Args):
             ))
         RunCommand (
             "GenFw --rebase 0x{:02X} -o {} {} ".format (
-              fit_image_info_header.LoadAddr + fit_image_info_header.DataOffset,
+              fit_image_info_header.LoadAddr,

```

Comment:
I think we also need to update load property in FIT header.
Could we create global variable so it will be easier to understand and maintain going forward?

for example:
#
# FD_BASE matching FDF
#
FdBase             = 0x00800000
FitHeaderLength    = 0x1000
LoadAddressDefault = FdBase + FitHeaderLength

line#247:  fit_image_info_header.DataOffset    = FitHeaderLength
line#333:     parser.add_argument('-l', "--LoadAddress", type=int, help='Specify payload load address', default =LoadAddressDefault)

Reply:
> I think we also need to update load property in FIT header.

I don't understand this, or there's a miscommunication. The entrypoint is now based to the load address by default. I have since updated the script so that the argument reflects EDK2-based systems will have their FIT header.

Reply:
my understanding is we want to support both scenarios:
1. PlatformInit loads UPL entry module without FIT header. PlatformInit then use FDT node to tell UPL entry what else child image should be loaded (good for the case where some child images are optional)
2. PlatformInit loads "FIT header + UPL entry module + all the child images" as a single raw binary always (in this case all child images are mandatory, nothing optional here)

So both "load" and "entry" properties of UPL entry module should not include FIT header size to support #1 case above. #2 PlatformInit will adjust to support it. Are we on the same page with this?